lib/logger.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+var Log;
+
+Log = require("log");
+
+module.exports = (args) => {
```

Comment:
There wasn't a need for it before, however, it looks like there has since been stuff added to it.